### DISCUSSION TITLE
Add confirmation step when downloading CSV files wit submissions

### DISCUSSION THREAD

**Discussion by frjo** [OpenTechFund/opentech.fund#4034] [General]
Downloading a csv is a serious security concern.

What do you think about adding a confirmation step (dialog box) to this action? a "Download" and a "Cancel" button with a short text about being carful with the file since it contains sensitive information.

**Comment by wes-otf** [comment]
yeah I agree, also because it's kinda resource intensive depending on what submissions are filtered (if any). A confirmation pop-up would make sure this was definitely intended.
